app/src/main/java/fr/neamar/kiss/db/DBHelper.java

```diff
@@ -212,11 +212,38 @@ public static void insertShortcut(Context context, ShortcutRecord shortcut) {
         db.insert("shortcuts", null, values);
     }
 
-    public static void removeShortcut(Context context, String name) {
+    public static void removeShortcut(Context context, String intentUri) {
         SQLiteDatabase db = getDatabase(context);
-        db.delete("shortcuts", "name = ?", new String[]{name});
+        db.delete("shortcuts", "intent_uri = ?", new String[]{intentUri});
     }
 
+    public static ArrayList<ShortcutRecord> getShortcuts(Context context, String packageName) {
+        ArrayList<ShortcutRecord> records = new ArrayList<>();
+        SQLiteDatabase db = getDatabase(context);
+
+        // Cursor query (String table, String[] columns, String selection,
+        // String[] selectionArgs, String groupBy, String having, String
+        // orderBy)
+        Cursor cursor = db.query("shortcuts", new String[]{"name", "package", "icon", "intent_uri", "icon_blob"},
+                "package LIKE ?", new String[]{"%" + packageName + "%"}, null, null, null);
```

Comment:
Do we need a LIKE here? I thought a simple `=` would be enough?

Reply:
Correct, a copy paste oversight.